src/popup/components/TokenAmount.vue

```diff
@@ -57,6 +57,7 @@ export default defineComponent({
     noSymbol: Boolean,
     highPrecision: Boolean,
     dynamicSizing: Boolean,
+    alignRight: Boolean,
```

Comment:
Why not using the CSS text-align on the parent level as this component is not a block but inline element?